demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -585,6 +586,20 @@ def test_handle_all_custom_content_flag(self, tmp_path):
             custom_content_names = [cco["name"] for cco in env.CUSTOM_CONTENT]
             assert ordered(custom_content_names) == ordered(downloader.input_files)
 
+    def test_handle_init_flag(self, tmp_path, mocker):
+        env = Environment(tmp_path)
+        mocker.patch.object(builtins, "input", return_value="test_pack_name")
+
+        downloader = Downloader(env.CONTENT_BASE_PATH, "")
+        downloader.init = True
+        downloader.handle_init_flag()
+
+        assert downloader.output_pack_path == str(
+            Path(env.CONTENT_BASE_PATH) / "Packs" / "test_pack_name"
+        )
+        assert Path(Path(downloader.output_pack_path) / "pack_metadata.json").exists()
+        assert not Path(Path(downloader.output_pack_path) / "Integrations").exists()
```

Comment:
please test `len(Path(downloader.output_pack_path ).iterdir() == ...` with the expected number of folders